<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学习使我快乐！</string>
  </property>
  <widget class="QWidget" name="mainwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lb_top">
      <property name="text">
       <string>每日励志标语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hMidLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lb_target">
          <property name="text">
           <string>今日学习目标</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="tx_target">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_progress">
          <property name="text">
           <string>今日完成进度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="tx_progress">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lb_down">
      <property name="text">
       <string>开始时间：学习时长：总学习时长：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuChart">
    <property name="title">
     <string>图像</string>
    </property>
    <addaction name="actionPie_Chart"/>
    <addaction name="actionLine_Chart"/>
    <addaction name="actionBar_Chart"/>
   </widget>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>输出</string>
    </property>
    <addaction name="actiongenerator"/>
    <addaction name="actionOpenIn_Explorer"/>
   </widget>
   <widget class="QMenu" name="menulabels">
    <property name="title">
     <string>标语</string>
    </property>
    <addaction name="actionOpenLabelsFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChart"/>
   <addaction name="menuOutput"/>
   <addaction name="menulabels"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>加载数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPie_Chart">
   <property name="text">
    <string>饼图</string>
   </property>
  </action>
  <action name="actionLine_Chart">
   <property name="text">
    <string>折线图</string>
   </property>
  </action>
  <action name="actionBar_Chart">
   <property name="text">
    <string>条形图</string>
   </property>
  </action>
  <action name="actiongenerator">
   <property name="text">
    <string>生成</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionOpenIn_Explorer">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actionOpenLabelsFile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
